recipe/meta.yaml

```
@@ -23,8 +21,6 @@ requirements:
   host:
     - pthreads-win32             # [win]
     - zlib
```

The [linking analysis in the Windows build](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=801417&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=6119ccbe-9301-594f-7c27-f792b80a7fcc&l=1763) appears to show it doesn't seem to link against these. I believe it might need to be this instead, since we're using the MinGW build stack (but not my expertise):

```suggestion
    - m2w64-zlib                 # [win]
    - zlib                       # [not win]
  run:
    - m2w64-winpthreads-git      # [win]
    - m2w64-zlib                 # [win]
```

---

Interesting that the MinGW packages don't have `run_exports`

Windows is not my strong suit. So am happy to defer to your judgement or save this to a future PR

---

Let's save it for a future PR.

---

Sounds good. Have filed issue ( https://github.com/conda-forge/pigz-feedstock/issues/21 ) to track. Please feel free to edit or add anything else there